topologies/datacenter-latest/labguides/source/cvp-vxlan.rst

```diff
@@ -89,12 +101,14 @@ VxLAN
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet.png
     :align: center
     :width: 50%
+|br|
 
 * Validate configlet syntax on **leaf3**
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet-validate.png
     :align: center
     :width: 50% 
+|br|
 
 4. Assign the VXLAN configlet to **leaf3**
```

Comment:
Should there be more bullet points explaining to right click on the switch, click manage -> configlet and then search for the configlet?